enhancements/cluster-logging/fluentd-tweaks.md

```diff
@@ -0,0 +1,166 @@
+---
+title: cluster-logging-log-forwarding
+authors:
+  - "@alanconway"
+reviewers:
+approvers:
+creation-date: 2020-05-27
+status: provisional
+see-also:[]
+replaces:[]
+superseded-by:[]
+---
+
+# Fluentd Peformance Tuning
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Naming of "tweaks"
+
+I've called these settings 'tweaks' (like the Gnome Tweaks application) to
+identify them clearly as things that most users don't need to set.  If there are
+objections to that, possible alternate names: 'knobs', 'hints', 'advanced',
+'expert', 'tuning' ... other?
```

Comment:
Isn't tuning more appropriate as that's what they are?   parameters  to tune in order to improve performance or reliability (e.g.  I tune the bass on my stereo because ...)

Reply:
I can live with that. Any other opinions?

Reply:
Done